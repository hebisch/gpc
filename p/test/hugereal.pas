program hugereal(input,output);
    var
        r : real;
        errno : integer;
    begin
        writeln('errno = ', errno);
        write('gimme a real ');
        readln(r);
        writeln('that was a really satisfying',r);
        writeln('errno = ', errno);
    end.
