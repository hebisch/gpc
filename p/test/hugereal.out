errno = 0
gimme a real that was a really satisfying 1.234567899876543e+08
errno = 0
